ui/app/src/utils/codebase-bridge.tsx

```diff
@@ -114,7 +114,9 @@ export function createCodebaseBridge() {
       AuthMonitor: lazy(() => import('../components/SystemStatus/AuthMonitor')),
       Login: lazy(() => import('../pages/Login')),
       BulkUpload: lazy(() => import('../components/BulkUpload')),
-      ConfirmDialog: lazy(() => import('../components/ConfirmDialog'))
+      ConfirmDialog: lazy(() => import('../components/ConfirmDialog')),
+      ClipDialog: lazy(() => import('@craftercms/video/Clipping/ClipDialog')),
```

Comment:
Looks like the ClipDialog is not used, only it’s wrapper (VideoPlayerDialog).

Reply:
In the Pr update it is being used;